timeseries/src/autogluon/timeseries/regressor.py

```diff
@@ -72,9 +78,11 @@ def __init__(
         self.max_num_samples = max_num_samples
         self.validation_fraction = validation_fraction
         self.fit_time_fraction = fit_time_fraction
+        self.include_static_features = include_static_features
+        self.include_item_id = include_item_id
 
         self.model: Optional[AbstractModel] = None
-        self.disabled_due_to_time_limit = False
+        self.disabled_during_inference = False
```

Comment:
Naming nit: why disabled "during inference"? Doesn't this mean that this regressor is just disabled, in general?

Reply:
Good point, just `disabled` then?